### PR TITLE
Rename Stonesoup back to AppStudio

### PR DESCRIPTION
**Pull request by arewm**
Also include the renaming of diagrams

### CODE REVIEW

**Comment by arewm** [comment]
I renamed all instances that I could find except for this one as I wasn't sure if it should be renamed as well:

https://github.com/redhat-appstudio/book/blob/52d1eed5941f55c5232e691dcc7d4412a1042912/ADR/0010-namespace-metadata.md?plain=1#L37

File: ADR/assets/0018-apps-continuous-perf-testing.svg
Comment (ralphbean) [inline ADR/assets/0018-apps-continuous-perf-testing.svg:1]: It looks like the svg didn't actually get updated here?

**Review by ralphbean — APPROVED** [review]

File: ADR/assets/0018-apps-continuous-perf-testing.svg
Comment (arewm) [reply ADR/assets/0018-apps-continuous-perf-testing.svg:1]: Yeah. I tried to do it locally but Inkscape seems to crash on my computer whenever I attempt to modify it.

**Comment by Michkov** [comment]
On 'RHTAP - Architecture Deep Dive' document there is AppStudio used 151x and App Studio used 134x

**Comment by arewm** [comment]
In https://github.com/redhat-appstudio/book/pull/34, it was AppStudio so that is the pattern that I followed to change it back to.

**Review by alexeykazakov — APPROVED** [review]

**Pull request merged**